test/functional/test_framework/test_framework.py

```diff
@@ -814,6 +814,9 @@ def set_genesis_mocktime(self):
         for node in self.nodes:
             node.mocktime = self.mocktime
 
+    def wait_until(self, test_function, timeout=60, lock=None):
```

Comment:
19552: partial?

Reply:
> Backport https://github.com/bitcoin/bitcoin/pull/19552 has only util releated code, because we do not have fee-filter message (and do not have plans to have it as far as I know)